.gitignore

```diff
@@ -1,4 +1,5 @@
 .project
 .idea
 charts/*/charts
-charts/*/Chart.lock
\ No newline at end of file
+charts/*/Chart.lock
+charts/*/activemq-artemis
```

Comment:
please remove this
 
you have to add in workflows as shown below
![image](https://user-images.githubusercontent.com/72004356/206384120-0ff1b222-f8e1-4d92-9776-a788b63da0c1.png)
![image](https://user-images.githubusercontent.com/72004356/206384258-ff0f65d5-e3e3-41c7-986a-2feaf016809f.png)

Reply:
update it on both helm-lint/yaml and helm-lint manual for develop and 1.2.0.1